src/Microsoft.Graph/Requests/Generated/SharedDriveItemRequest.cs

```diff
@@ -136,6 +137,30 @@ public ISharedDriveItemRequest Expand(string value)
         }
 
         /// <summary>
+        /// Adds the specified expand value to the request.
+        /// </summary>
+        /// <param name="expandExpression">The expression from which to calculate the expand value.</param>
+        /// <returns>The request object to send.</returns>
+        public ISharedDriveItemRequest Expand(Expression<Func<SharedDriveItem, object>> expandExpression)
```

Comment:
There are changes in the templates. microsoftgraph/MSGraph-SDK-Code-Generator#80
The level of duplication in the client libraries is less than in the templates. There should be little duplication in the templates. It would require refactoring the templates to produce and use base classes to remove the duplicate functions -- this is unlikely to happen.